src/components/Field.vue

```diff
@@ -1,16 +1,19 @@
 <template>
-  <div class="form-group">
-    <label v-if="label">{{ label }}</label>
-    <p v-if="description">{{ description }}</p>
-    <input v-if="type === 'text'" type="text" class="form-control" v-model="value" :placeholder="placeholder" :required="required">
-    <input v-if="type === 'email'" type="email" class="form-control" v-model="value" :placeholder="placeholder" :required="required">
-    <textarea v-if="type === 'textarea'" class="form-control" v-model="value" :placeholder="placeholder" :required="required" :rows="rows || 4"></textarea>
-    <div class="options" v-if="options && ['radio', 'checkbox'].indexOf(type) !== -1">
-      <label v-bind:key="option.value || option" v-for="option in options">
-        <input :name="name" :type="type" :value="option.value" /> {{ option.label || option.value || option }}
-      </label>
+  <div>
```

Comment:
Why the additional layer of `<div>`? The field could use the label and description etc. as well, so it seems to make sense for it to be part of the `form-group`.

Reply:
@tonytlwu I initially added when developing via the CLI (as the form group seemed to have a padding which the iframe already has internally), but this doesn't seem to be necessary in Studio so I've now removed it.

Reply:
![image](https://user-images.githubusercontent.com/574210/107966381-0b535200-6fac-11eb-916c-2c0163b96821.png)